src/dbManager.php

```diff
@@ -23,12 +23,47 @@ public function writedb($sites, $timestamp) {
         $this->pdo->query($query);
     }
 
+    /**
+     * @return SQLite3Result
+     */
     public function readDB() {
```

Comment:
I'd call this method something like `getResults()` since that's what the 'contract' is.  Reading the DB is an implementation detail.